api/app/wildfire_one.py

```diff
@@ -132,25 +146,37 @@ def _is_station_valid(station) -> bool:
     return True
 
 
+def fetch_ecodivision_name(latitude: str, longitude: str, ecodivisions: geopandas.GeoDataFrame):
```

Comment:
Minor: Does this function make any I/O or network calls to get ecodivisions? The verb `fetch` maps to I/O call in my head (maybe just me?) but I don't see any obvious ones here.

Reply:
Changed the name to `get_ecodivision_name` :)